### PR TITLE
New api updates

### PR DESCRIPTION
**Pull request by anjuligoring**
- adding new windows images
- including new sku sizes
- updating defaults

### CODE REVIEW

File: parts/params.t [hunk 1]
```diff
@@ -51,18 +51,19 @@
...
+        "Standard_LRS"
       ],
       "metadata": {
         "description": "Type of managed disk to create."
       }
     },
     "securityType": {
       "type": "string",
-      "defaultValue": "Unencrypted",
+      "defaultValue": "VMGuestStateOnly",
```
Comment (yentsanglee) [inline parts/params.t:66]: >VMGuestStateOnly [](http://example.com/codeflow?start=23&length=16)

Run suggested "DiskWithVMGuestState" as default.

File: parts/vars.t [hunk 2]
```diff
@@ -1,27 +1,27 @@
...
       },
       "Ubuntu 20.04 LTS Gen 2": {
-        "publisher":  "AZURERT.PIRCORE.CAPSBVT",
-        "offer":  "longlivedconfidentialvm",
-        "sku":  "Ubuntu20.04",
-        "version": "0.0.1"
+        "publisher": "Canonical",
+        "offer": "0001-com-ubuntu-confidential-vm-experimental",
+        "sku": "20_04-gen2",
+        "version": "20.04.20210309"
```
Comment (yentsanglee) [inline parts/vars.t:18]: >"20.04.20210309" [](http://example.com/codeflow?start=19&length=16)

Why is this not "latest"?

File: parts/vars.t [hunk 3]
```diff
@@ -1,27 +1,27 @@
...
       },
       "Ubuntu 18.04 LTS Gen 2": {
-        "publisher":  "AZURERT.PIRCORE.CAPSBVT",
-        "offer":  "longlivedconfidentialvm",
-        "sku":  "Ubuntu18.04",
-        "version": "0.0.1"
+        "publisher": "Canonical",
+        "offer": "0001-com-ubuntu-confidential-vm-experimental",
+        "sku": "18_04-gen2",
+        "version": "18.04.20210309"
```
Comment (yentsanglee) [inline parts/vars.t:24]: >"18.04.20210309" [](http://example.com/codeflow?start=19&length=16)

Why is this not "latest"?

**Review by yentsanglee — APPROVED** [review]
:shipit:

File: parts/vars.t (hunk 3 above)
Comment (anjuligoring) [reply parts/vars.t:24]: these are the versions that qiucheng sent me that were validated, i haven't tried with 'latest' yet, but do you think that would be better or could cause issues if the latest release has issues in it ?

Comment (anjuligoring) [reply parts/vars.t:24]: these are the versions that qiucheng sent me that were validated, i haven't tried with 'latest' yet, but do you think that would be better or could cause issues if the latest release has issues in it ?

**Pull request merged**